config/deploy.rb

```diff
@@ -0,0 +1,42 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.19.2"
+
+set :application, "nztrain"
+set :repo_url, "git@github.com:NZOI/nztrain.git"
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+
+# Default deploy_to directory is /var/www/my_app_name
+# set :deploy_to, "/var/www/my_app_name"
+
+# Default value for :format is :airbrussh.
+# set :format, :airbrussh
+
+# You can configure the Airbrussh format using :format_options.
+# These are the defaults.
+# set :format_options, command_output: true, log_file: "log/capistrano.log", color: :auto, truncate: :auto
+
+# Default value for :pty is false
+# set :pty, true
+
+# Default value for :linked_files is []
+append :linked_files, "config/database.yml", "config/redis.yml"
```

Comment:
I think we'll also need `config/isolate.yml`, `config/secret_key_base.production.txt`, `script/install.cfg` to persist between releases. There's also a few other config files listed in .gitignore that could be worth adding, though I think we don't currently use them.

Also I think assets under `db/data` should go under linked_dirs?

Reply:
I have no idea what's going on with db/data is is spooking me out - added the rest though.